redis/src/main/scala/zio/redis/transactional/Redis.scala

```diff
@@ -0,0 +1,14 @@
+package zio.redis.transactional
+
+import zio.redis.RedisExecutor
+import zio.schema.codec.BinaryCodec
+import zio.{URLayer, ZLayer}
+
+trait Redis extends api.Sets
```

Comment:
This seems to duplicate the entire API, do we really need it? Can't we have transactional combinators on top of the existing API and use transactions as "scoping" mechanism?